### ISSUE TITLE
Inconsistent order of track artists

### ISSUE THREAD

**Issue by jaedb** [mopidy/mopidy-spotify#169]
Fetch a track (eg `spotify:track:1tRoJfwQ2upkIVEVKkuPcT`) from the Spotify HTTP API and the artist order differs from that returned when requesting the same track via the Mopidy-Spotify backend.

**To replicate:**
1. Get the track using the Spotify Console ([link](https://beta.developer.spotify.com/console/get-track/?id=1tRoJfwQ2upkIVEVKkuPcT&market=)).
2. Observe the order of artists: _ODESZA, WYNNE, Mansionair_.
3. Get the track from Mopidy-Spotify (I use _Settings > Debug_ tool in Mopidy-Iris), eg `library.lookup` `{"uri":"spotify:track:1tRoJfwQ2upkIVEVKkuPcT"}`.
4. Notice the order of artists differs: _WYNNE, Mansionair, ODESZA_.

Are there any ordering rules applied by Mopidy-Spotify when the array of artists is returned? Or perhaps caching of artists affects the return? Let me know if I can do any further testing to help identify the issue.

**Comment by kingosticks** [comment]
Nope, no ordering by us. They use two different backends at Spotify. I think the Web API orders some things by (worldwide) popularity, is that what you see? Not sure what libspotify's backend does but it's not necessarily the same.

**Comment by jodal** [comment]
> *This was generated by AI during triage.*

Mopidy-Spotify 5 gets all track metadata from the Spotify Web API, so the order of artists is now the same as the Web API returns. Closing as completed.
